simulations/CC_cycle.py

```diff
@@ -91,14 +93,23 @@ def terminate_check(t, SV, SVdot, return_val, inputs):
         i_data = currents[i]*np.ones_like(solution.values.t)
         cycle_number = int(i+1-equil)*np.ones_like(solution.values.t)
         cycle_capacity = 0.1*solution.values.t*abs(i_data)/3600
+        voltage = solution.values.y.T[-7]
+        power_density = voltage*i_data
```

Comment:
This should be a single number in the end, which is a time-weighted average of each of these values.
Also, I think you can do the work below without a loop, by just slicing the arrays.

```python
delta_t = solution.values.t[1:] = solution.values.t[:-1]
energy_density = np.sum(voltage[1:],delta_t)*currents[I]/3600 # should be mAh/cm2, but please check my math
power_density = 3600*energy_density / solution.values.t[-1] #Again, check my unit conversion.
```

One question is: how do we want to store/save these? It doesn't really fit in the array of solution vectors. 

Maybe write a txt file or a separate csv that has metrics like this?